tests/routers/test_remote.py

```
@@ -1,99 +1,76 @@
-        db_session.refresh(job)
-        return job
-    return _factory
+
+# Valid fake data for schema validation
+VALID_FINGERPRINT = "a" * 64
+VALID_PUBKEY = base64.b64encode(b"0" * 32).decode("utf-8")
+
+
+# Helper class for mocking aiofiles.open context manager
+class MockAsyncFile:
+    def __init__(self):
```

Ruff (0.15.1)

[warning] Missing return type annotation for special method `__init__`

Add return type annotation: `None`

(ANN204)